technical-reports/format/groups.md

```diff
@@ -110,9 +134,10 @@ For example:
 ```json
 {
   "brand": {
-    "type": "color",
+    "metadata": {
+      "type": "color"
```

Comment:
I like this. This allows someone to both set a default type for this group, as well as also create a `type` subgroup, which wouldn’t be out-of-the-question for typography tokens.

Reply:
Good point! i hadn't actually considered that before.

In effect, this means that only `value` and `metadata` need to be reserved words. The reason is that `value` is the only required prop in a token and therefore the way in which parsers can differentiate between a group and a token.

I.e. if an object has a property called  `value`, it must be a token. If not, ituat be a group. Consequently, groups cannot contain items called `value`.

Reply:
Yeah, name clashing crossed my mind when I read this spec today, I like this change as well!

Reply:
To expand on this: I’m currently working on a [parser for design tokens](https://github.com/drwpow/cobalt-ui) and the `type` key is a really important part of the schema, both for parsers as well as type systems like TypeScript to infer which is which. Having both a group and token possess a top-level `type: "color"` key would make everything exponentially harder/slower to reason about. So for that reason, I do like it being `metadata` on the group.

If group did have `type`, it’d make things easiest for parsers to be `type: "group"`. But that would probably cause confusion with the `metadata.type` proposal here, and overall that wouldn’t add _that_ much value for parsers as it can be inferred.